samples/kubernetes/deploy-aks.sh

```diff
@@ -0,0 +1,214 @@
+#!/bin/bash
+
+set -eE -o functrace
+
+failure() {
+  local lineno=$1
+  local msg=$2
+  echo -e "\033[31mScript failed at $lineno: ${msg}\e[0m"
+}
+
+trap 'failure ${LINENO} "$BASH_COMMAND"' ERR
+
+usage() {
+    echo "deploy-aks.sh  --resource-group-name <group name>"
```

Comment:
out of sync with actual parsing

Reply:
Switch to name and resource group name (which defaults to name if not provided)